demo/12px/index.js

```diff
@@ -13,6 +13,8 @@ Garden.svgIDs = [
   'zd-svg-icon-12-align-right-stroke',
   'zd-svg-icon-12-arrow-left-fill',
   'zd-svg-icon-12-arrow-left-stroke',
+  'zd-svg-icon-12-arrow-retweet-fill',
```

Comment:
Should this be a "branded" name?

Reply:
Good question. In general, we want to name icons what they "are" vs. what they "do". @rossmoody and I had a discussion about this and came to this compromise.